client/lib/feed-stream-store/feed-stream.js

```diff
@@ -519,6 +536,22 @@ export default class FeedStream {
 			} );
 		}
 
+		// if conversations, then unleash the tucked away comments to redux
+		if ( !! get( this.pendingPostKeys, '0.comments' ) ) {
+			forEach( this.pendingPostKeys, postKey => {
+				const key = postKeyToString( postKey );
+				const comments = this.pendingComments.get( key );
+				this.pendingComments.delete( key );
+
+				reduxDispatch( {
+					type: COMMENTS_RECEIVE,
+					siteId: postKey.blogId,
+					postId: postKey.postId,
+					comments,
+				} );
+			} );
```

Comment:
should we reset the pendingComments map here too?

Reply:
or maybe down on line 564 where we reset the pendingPostKeys

Reply:
i manually delete the key after use, so theoretically this should never need to be cleared.
I'll add a map.clear under the pendingPostKeys = [] on 564 👍

Reply:
done